ui-events/CHANGELOG.md

```diff
@@ -17,19 +17,29 @@ This release has an [MSRV][] of 1.73.
 
 ### Added
 
-* `PointerId`, `PointerInfo`, `PointerUpdate`, and `PointerEvent` now have a `is_primary_pointer` method. ([#54][] by [@waywardmonkeys][])
+* `PointerId`, `PointerInfo`, `PointerUpdate`, and `PointerEvent` now have an `is_primary_pointer` method. ([#54][] by [@waywardmonkeys][])
+* `PointerGesture` and `PointerGestureEvent` types, with `Gesture` variant added to `PointerEvent`. ([#80][] by [@xorgy][] and [@arthur-fontaine][])
 
-## [0.1.0][] - 2025-05-08
```

Comment:
This seems wrong.

Reply:
It is moved in the diff, diff just looks funny I think...

Reply:
Moved things around in the diff to make it clear that this was unchanged.